packages/drawer/src/Drawer/Drawer.tsx

```diff
@@ -18,27 +19,30 @@ export const Drawer = forwardRef<HTMLDivElement, DrawerProps>(
     {
       children,
       className,
-      'data-lgid': dataLgId = LGIDs.root,
+      'data-lgid': dataLgId = DEFAULT_LGID_ROOT,
```

Comment:
Do you get a TS error here? it's expecting `lg-${string}`.

Reply:
We could maybe prepend the `lg` to the root if we don't want to keep having to type `lg`